api/receiver/receive_results_test.go

```diff
@@ -7,18 +7,41 @@
 package receiver
 
 import (
+	"fmt"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
 	"os"
+	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
+	"github.com/web-platform-tests/wpt.fyi/shared"
 	"google.golang.org/appengine/taskqueue"
 )
 
+type regexMatcher struct {
```

Comment:
Can we get a comment on the rationale for implementing a custom matcher? I'm sure reading the code eventually reveals it, but it would be helpful to have it up front.